source/administration/mobile-changelog.md

```diff
@@ -1,5 +1,39 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.42.0 Release
+- Release Date: April 16, 2021
+- Server Versions Supported: Server v5.25+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.31 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.25 has ended and upgrading to server ESR v5.31 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Improvements
+ - Apps Framework
```

Comment:
```suggestion
 - Support for the new Apps Framework (Developer Preview)
```

Reply:
Keep in mind: This feature is not usable on Mobile until the App Framework actually becomes available on webapp and the proxy plugin is loaded on an instance.  That's why I said "support".  The launch for the developer preview of the Apps Framework is at the end of this month (April 29th).